proc_install-knative-OCP-4x.adoc

```diff
@@ -0,0 +1,48 @@
+// This assembly is included in the following assemblies:
+//
+// assembly_knative-OCP-4x.adoc
+
+= Installing Knative on an OpenShift cluster 4.0 using the script provided
+
+NOTE: This Knative on OpenShift preview is only available by using the OpenShift 4.0 developer preview. 
+
+.Prerequisites
+* You will require a Red Hat Developers login to try this. 
+
+
+.Procedure 
+
+. Login to the cluster using your admin credentials.
+
+   `oc login <admin-credentials>`
+   
+. Clone the `knative-operators` repository.
+
+   `git clone https://github.com/openshift-cloud-functions/knative-operators`   
+   `cd knative-operators/`   
+   `git fetch --tags`   
+   `git checkout openshift-v0.2.0`   

```

Comment:
v0.2.0 needs to be updated but I'm not sure what the new tags are - @bbrowning is providing some info at the minute for any additional changes needed to the OCP 4.0 docs for the 0.3.0 release.